Cypress tests fail

### Describe the bug

Seems like after #206 E2E tests fail with an error

### Steps to Reproduce

https://github.com/10up/autoshare-for-twitter/actions/runs/3658537966/jobs/6251228057

### Screenshots, screen recording, code snippet

```
Run npm run cypress:run

> @[10](https://github.com/10up/autoshare-for-twitter/actions/runs/3658537966/jobs/6251228057#step:9:11)up/autoshare-for-twitter@1.2.1 cypress:run
> cypress run --config-file tests/cypress/cypress.config.js

The cypress npm package is installed, but the Cypress binary is missing.

We expected the binary to be installed here: /home/runner/.cache/Cypress/[11](https://github.com/10up/autoshare-for-twitter/actions/runs/3658537966/jobs/6251228057#step:9:12).2.0/Cypress/Cypress

Reasons it may be missing:

- You're caching 'node_modules' but are not caching this path: /home/runner/.cache/Cypress
- You ran 'npm install' at an earlier build step but did not persist: /home/runner/.cache/Cypress

Properly caching the binary will fix this error and avoid downloading and unzipping Cypress.

Alternatively, you can run 'cypress install' to download the binary again.

https://on.cypress.io/not-installed-ci-error

----------

Platform: linux-x64 (Ubuntu - [22](https://github.com/10up/autoshare-for-twitter/actions/runs/3658537966/jobs/6251228057#step:9:23).04)
Cypress Version: 11.2.0
Error: Process completed with exit code 1.
```

### Environment information

_No response_

### WordPress information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.github/workflows/build-release-zip.yml
+++ b/.github/workflows/build-release-zip.yml
@@ -32,7 +32,9 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: node_modules
+          path: |
+            - node_modules
+            - /home/runner/.cache/Cypress
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
       - name: Setup node version and npm cache

--- a/.github/workflows/build-release-zip.yml
+++ b/.github/workflows/build-release-zip.yml
@@ -33,8 +33,9 @@ jobs:
           cache-name: cache-node-modules
         with:
           path: |
-            - node_modules
-            - /home/runner/.cache/Cypress
+            node_modules
+            ~/.cache
+            ~/.npm
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
       - name: Setup node version and npm cache

--- a/.github/workflows/cypress.yml
+++ b/.github/workflows/cypress.yml
@@ -49,8 +49,9 @@ jobs:
           cache-name: cache-node-modules
         with:
           path: |
-            - node_modules
-            - /home/runner/.cache/Cypress
+            node_modules
+            ~/.cache
+            ~/.npm
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
       - name: Install dependencies

--- a/.github/workflows/cypress.yml
+++ b/.github/workflows/cypress.yml
@@ -8,10 +8,10 @@ on:
   pull_request:
     branches:
       - develop
-    paths:
-      - '**.js'
-      - '**.php'
-      - '**.css'
+    # paths:
+    #   - '**.js'
+    #   - '**.php'
+    #   - '**.css'
 
 jobs:
   build:

--- a/.github/workflows/cypress.yml
+++ b/.github/workflows/cypress.yml
@@ -8,10 +8,10 @@ on:
   pull_request:
     branches:
       - develop
-    # paths:
-    #   - '**.js'
-    #   - '**.php'
-    #   - '**.css'
+    paths:
+      - '**.js'
+      - '**.php'
+      - '**.css'
 
 jobs:
   build:

--- a/.github/workflows/cypress.yml
+++ b/.github/workflows/cypress.yml
@@ -48,7 +48,9 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: node_modules
+          path: |
+            - node_modules
+            - /home/runner/.cache/Cypress
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
 
       - name: Install dependencies

